File: Zad3/app/src/main/java/com/example/bmizad2/ui/bmiCalculator/BMICalculatorViewModel.java
package com.example.bmizad2.ui.bmiCalculator;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class BMICalculatorViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public BMICalculatorViewModel() {
        mText = new MutableLiveData<>();

        mText.setValue("Enter weight and height to count BMI");

    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: Zad4/app/src/main/java/com/example/bmizad2/ui/bmiCalculator/BMICalculatorViewModel.kt
package com.example.bmizad2.ui.bmiCalculator

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class BMICalculatorViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()

    init {
        mText.value = "Enter weight and height to count BMI"
    }

    val text: LiveData<String>
        get() = mText
}